clusterloader2/pkg/provider/kubemark.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"fmt"
+	sshutil "k8s.io/kubernetes/pkg/ssh"
+)
+
+const KubemarkName = "kubemark"
+
+func init() {
+	getProviderCtorMap()[KubemarkName] = NewKubemarkProvider
+}
+
+type KubemarkProvider struct {
+	features Features
+	config   Config
+}
+
+func NewKubemarkProvider(config map[string]string) Provider {
+	supportEnablePrometheusServer := true
+	if config[RootKubeConfigKey] == "" {
+		fmt.Println("no kubemark-root-kubeconfig path specified. SupportEnablePrometheusServer will be false.")
```

Comment:
we generally don't use fmt.Println.

Please use klog package instead (klog.Warningf(...))